hudi-client/src/test/java/org/apache/hudi/TestAsyncCompaction.java

```diff
@@ -248,8 +238,7 @@ public void testScheduleIngestionBeforePendingCompaction() throws Exception {
     metaClient = new HoodieTableMetaClient(jsc.hadoopConfiguration(), cfg.getBasePath());
     HoodieInstant pendingCompactionInstant =
         metaClient.getActiveTimeline().filterPendingCompactionTimeline().firstInstant().get();
-    assertTrue("Pending Compaction instant has expected instant time",
-        pendingCompactionInstant.getTimestamp().equals(compactionInstantTime));
+    assertEquals("Pending Compaction instant has expected instant time", pendingCompactionInstant.getTimestamp(), compactionInstantTime);
 
```

Comment:
line 246 `HoodieTableMetaClient metaClient = new HoodieTableMetaClient(jsc.hadoopConfiguration(), cfg.getBasePath());` would be removed too?

Reply:
done.